modules/administration-guide/partials/con_external-postgresql-setup.adoc

```diff
@@ -0,0 +1,34 @@
+[id="external-postgresql-setup_{context}"]
+= External PostgreSQL setup
+
+By default, {prod-short} Operator creates and manages deployment of the internal PostgreSQL database. However, {prod-short} Operator does not support full life-cycle capabilities such as backups and recovery.
+
+An external PostgreSQL database is recommended for a business-critical setup. The external PostgreSQL can be configured for High Availability (HA) and Point In Time Recovery (PITR).
+
+The external PostgreSQL instance can be configured as on-premises or to use a cloud service such as Amazon Relational Database Service (Amazon RDS). Amazon RDS, for example, enables deployment of production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
+
+The recommended configuration of an example database:
+
+[options="header"]
+|===
+|Parameter | Value
+|Instance class | db.t2.small
+|vCPU | 1
+|RAM | 2 GB
+|Multi-az | true, 2 replicas
+|Engine version | 9.6.11
+|TLS | enabled
+|Automated backups | enabled (30 days)
+|===
+
+Making workspace metadata and user information persistent involves configuring the external PostgreSQL database and configuring Che to use the configured external PostgreSQL database:
```

Comment:
```suggestion
You can make workspace metadata and user information persistent by configuring the external PostgreSQL database and configuring Che to use the configured external PostgreSQL database:
```
- Re-word to make more user-centric.
- Should `Che` be a variable like `{prod-short}`?

Reply:
@bburt-rh, thank you and yes.